app/src/main/java/com/sun_asterisk/moviedb_50/screen/home/HomePresenter.kt

```diff
@@ -0,0 +1,43 @@
+package com.sun_asterisk.moviedb_50.screen.home
+
+import com.sun_asterisk.moviedb_50.data.model.Genres
+import com.sun_asterisk.moviedb_50.data.model.Movie
+import com.sun_asterisk.moviedb_50.data.source.remote.OnFetchDataJsonListener
+import com.sun_asterisk.moviedb_50.data.source.remote.Repository
+
+class HomePresenter(private val mRepository: Repository?) : HomeContract.Presenter {
```

Comment:
repository sao null được 
```suggestion
class HomePresenter(private val repository: Repository) : HomeContract.Presenter {
```